adam-cli/src/main/scala/org/bdgenomics/adam/cli/FlagStat.scala

```diff
@@ -121,7 +121,8 @@ class FlagStat(protected val args: FlagStatArgs) extends BDGSparkCommand[FlagSta
         val fs = FileSystem.get(conf)
         val path = new Path(outputPath)
         val outputStream = fs.create(path, true)
-        outputStream.writeUTF(output)
+        outputStream.writeBytes(output)
```

Comment:
if all output chars are < 128 (which should always be true of this output), then `writeBytes` should actually be UTF8, whereas `writeChars` will output two bytes per char, and the former seems preferable to me? I was thinking that the existing `println` behavior is just doing one byte per char but I don't actually know, this is not my area of expertise. lmk what you think.